libclamav/ole2_extract.c

```diff
@@ -1295,7 +1295,7 @@ static cl_error_t handler_enum(ole2_header_t *hdr, property_t *prop, const char
                     if (prop->size < (int64_t)hdr->sbat_cutoff) {
                         if (!ole2_get_sbat_data_block(hdr, hwp_check, prop->start_block)) {
                             status = CL_EREAD;
-                            break;
+                            goto done;
```

Comment:
I don't think we want to do this. I think the correct solution is to remove `status = CL_EREAD;` instead.